roles-in-tech/sales-engineering.md

```diff
@@ -2,91 +2,100 @@
 
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+- Lesson: 25 min
+- Guided Practice: 10 min
+- Independent Practice: 4 min
+- Check for Understanding: 10 min
 
 ### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
+NOne
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+- Technology has advanced in leaps and bounds in recent years, and continues to. But not everyone has the time or know-how to keep up with these advancements. 
+- The role of sales engineers involves translating and explaining highly complex technical information to customers and clients, focusing on revealing how a product or piece of equipment can solve specific problems. 
+-  They therefore play an important role in the modern sales process. 
 
 ### Objectives
 
 **Participants will be able to:**
 
-- Things
-- More things
-- Even more things
-- Even more things
+- Understand the role of a sales engineer in a company
+- Decide whether they are suitable for a future as a sales engineer
+
 
 ### Specific Things To Teach
 
-- Things about the things
-- More things about the things
-	- This is a sub-thing about the things
-- Even more things about the things
-- Even more things about the things
+- Who is a sales engineer
+- Duties of a sales engineer
+- Traits of a good sales engineer
+- Steps to take for a career as a sales engineer
 
 ### Materials
 
```

Comment:
Just to double check: 
1. Is everything that is assigned in Lesson, Practices or Challenge listed in the materials section?
2. Is there anything in the materials section that is not assigned, but is just additional?  If so, move it to supplemental Materials.
3. Is there anything in supplemental materials that is assigned? then move it to "materials".